irobot_benchmark/CMakeLists.txt

```diff
@@ -33,12 +33,89 @@ set(TOPOLOGY_FILES
   topology/debug_sierra_nevada_best_effort.json
   topology/debug_mont_blanc_reliable.json
   topology/debug_mont_blanc_best_effort.json
+  topology/debug_white_mountain_best_effort.json
```

Comment:
Out of curiosity, what is the purpose/goal of the `debug_*_best_effort.json` topologies?

Reply:
It's a topology with a single node that subscribes to all topics published by the white-mountain.
it simulates a "debug station" connecting to the robot for logging or visualization.

FWIW, these debug topologies could be automatically generated